lib/livereload.js

```diff
@@ -97,6 +97,12 @@ function StartServer() {
                         return monkeyPatch() + snippet + match;
                     }
                 }
+            },
+            minify: false,
```

Comment:
Well, it's documentation. If you were new to the codebase, having defaults explicitly mentioned would give you a better picture of things. I usually like explicit things instead of things working magically for me, of course, there are cases where this is might not be applicable.